eclipse/src/saros/editor/annotations/SarosAnnotation.java

```diff
@@ -73,31 +78,46 @@ public static Color getUserColor(int colorID) {
   }
 
   /**
-   * Returns the color that corresponds to a user.
+   * Returns the color for the given annotation type and user color id.
    *
    * <p><b>Important notice:</b> Every returned color instance allocates OS resources that need to
    * be disposed with {@link Color#dispose()}!
    *
    * @param type
```

Comment:
Complete javadoc for params now that method is public?

Reply:
TBH adding Javadoc will not help you. The type numbers are hardcoded in the plugin.xml file and also hardcoded into the code.

Reply:
Yes, but you could point out that `Annotation.getType()` (of the appropriate annotation implementation) should be used to obtain the type as it will give you the only valid options.